modules/core/test/v2/unit/keychains.ts

```diff
@@ -40,6 +42,41 @@ describe('V2 Keychains', function v2keychains() {
     }));
   });
 
+  /**
+   * This section's intention is to provide some key generation sanity checking. generateKeyPair is a general surface
+   * for key generation but the keys are treated the same by BitGo down the line. Any SECP256K1 based coins key-pairs can
+   * be re-used so need to be the same.
+   **/
+  describe('Key generation enforcement for SECP256K1', function() {
+    // iterate over non-fiat crypto secp coins
+    const coinFamilyValues = Object.keys(CoinFamily).map(n => n.toLowerCase());
+    const cryptoSecpCoins = coins.filter(n => n.primaryKeyCurve === SupportedKeyCurve.Secp256k1
+      && n.kind === CoinKind.CRYPTO
+      && n.asset !== UnderlyingAsset.USD
+      && coinFamilyValues.includes(n.name));
+
+    const expectedXpub = 'xpub661MyMwAqRbcGpZf8mxNWhSPdWaLGvQzzage6vq2oQFzq8toVzmkjygYZ3HcZw6eCzAfn9ZdyGjKoKkcpKwackdgznVbiunpq7rkxDu7quS';
+    const expectedXprv = 'xprv9s21ZrQH143K4LVC2kRN9ZVf5UjqsTh9dMm3JYRRF4j1xLZexTTWCBN4hmdZUHeT3vCJPL181ErVaY489ArBKSWaB7Du7vyVS6XC43WtK7A';
+
+    const seed = Buffer.from('this is some random seed we will use', 'utf-8');
+
+    cryptoSecpCoins.forEach(function(coinName) {
+      before(function() {
+      });
+
+      it('should create the same key with the same seed for coin ' + coinName.name, function() {
```

Comment:
I'd prefer if you move the loop inside the `it` instead of the other way around. This breaks intellij mocha integration.

Reply:
ok done